tools/csv-merger/csv-merger.go

```diff
@@ -463,6 +463,19 @@ The OCS operator is the primary operator for Red Hat OpenShift Container Storage
         },
         "spec": {
             "manageNodes": false,
+            "monPVCTemplate": {
```

Comment:
fyi, anything that isn't generated by the `operator-sdk olm-catalog gen-csv` command is injected by this csv-merger.go binary as all the CSVs are being merged together. Things like the ocs-operator CSV description and alm-examples fall into this category.